common/protob/messages-bitcoin.proto

```diff
@@ -170,6 +180,7 @@ message SignTx {
     optional uint32 version_group_id = 8;               // only for Zcash, nVersionGroupId
     optional uint32 timestamp = 9;                      // only for Peercoin
     optional uint32 branch_id = 10;                     // only for Zcash, BRANCH_ID
+    optional AmountUnit amount_unit = 11;               // show amounts in
```

Comment:
let's set `[default=BITCOIN]` so that we don't need to deal with case this is not sent

Reply:
Fix in 3a1b98050463c515587349cf6abbb4090d6ee8ab